scripts/install-gems

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Can we use `/usr/bin/env bash` here, and ensure we do `set -e` below, so it fails with the right exit code

Reply:
I don't want this to be tied to `bash`, so I'd rather leave the shebang alone. I'll change the line below to `exec` though so that we get the right exit code behaviour.

Reply:
d02b345